### ISSUE TITLE
Toolbox items for text accents

### ISSUE THREAD

**Issue by ktraunmueller** [ktraunmueller/Compositor#48]
```
\DeclareTextAccentDefault{\"}{OT1}	umlaut, dieresis
\DeclareTextAccentDefault{\'}{OT1}	acute
\DeclareTextAccentDefault{\.}{OT1}	dot over
\DeclareTextAccentDefault{\=}{OT1}	macron
\DeclareTextAccentDefault{\H}{OT1}	hungarian umlaut
\DeclareTextAccentDefault{\^}{OT1}	circumflex
\DeclareTextAccentDefault{\`}{OT1}	grave
\DeclareTextAccentDefault{\b}{OT1}	underbar
\DeclareTextAccentDefault{\c}{OT1}	cedilla: fa\c{c}ade
\DeclareTextAccentDefault{\d}{OT1}	dot under
\DeclareTextAccentDefault{\r}{OT1}	ring over
\DeclareTextAccentDefault{\u}{OT1}	breve
\DeclareTextAccentDefault{\v}{OT1}	caron, háček
\DeclareTextAccentDefault{\~}{OT1}	tilde
```

**Comment by ktraunmueller** [comment]
Splitting this up into several tickets (accents, basic LaTeX symbols, etc).

**Comment by ktraunmueller** [comment]
**Implemented** in 0.9.17
